### ISSUE TITLE
No pokemons spawning.

### ISSUE THREAD

**Issue by GabrielTK** [maierfelix/POGOserver#241]
Good Evening!
I managed to run this server with my PR #238, but I can't spawn any pokemon through the Web API. It doesn't matter if I put my e-mail or my username at the username on console. I also tried to use "Spawnpoints", but nothing. I am trying to spawn an `Articuno` (exactly as I wrote here).
I often See this messages:
`(node:13524) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 26): TypeError: Cannot read property 'toUpperCase' of undefined
[Console] Spawned 1x undefined:17 at 10722780925956980736`
Can anyone help me please??
Thank you,
Gabriel

**Comment by nliakm** [comment]
I'm pretty sure, that the spawn command is not working atm.
but the spawnpoints must be working.

**Comment by Cronick** [comment]
@nliakm is right. The /spawn command is not working atm.
Only spawnpoints.

**Comment by d0dUxDJ** [comment]
I suggest you do the following:

1) Open and login to your webapi;
2) Create a spawnpoint via double clicking -> Interval = 60; Encounters = 144 (number for Articuno);
3) Restart the server (not completely sure if it is necessary, pretty sure it is though).

Check your database, you should have a working spawnpoint now. Check if it works ingame too. That worked for me.

**Comment by GabrielTK** [comment]
@d0dUxDJ It Works, but Isn't there any other way without restarting anything???

**Comment by d0dUxDJ** [comment]
I don't think so, Gabriel. When and if the spawn command will be fixed, it will be possible again, probably. We just have to wait and hope :)

**Comment by Badley187** [comment]
you dont need to restart the server after adding spawnpoints or pokestops. wait some seconds... maybe 30.
it worked without restart

**Comment by GabrielTK** [comment]
But... Whats the problem with the problem with the spawn command??? Let me see if we can fix it

**Comment by Cronick** [comment]
The command got removed a few updates ago, cause of structure changes
